### DISCUSSION TITLE
NO Transmitted Power

### DISCUSSION THREAD

**Discussion by SatasOdin** [Anime4000/RTL960x#109] [Q&A]
Sorry, I'm writing through a translator.
Purchased DFP-34X-2 C2 , software M110_sfp_ODI_220817. FiberHome HG6245D provider equipment .

In the FiberHome HG6245D itself, when the LOID\POLAM is changed, the provider is still authorized. Therefore, I think that authorization goes by GPON SN.

- <html>
<body>
<ul style="box-sizing: border-box; list-style-type: disc; color: rgb(53, 60, 65); font-family: Inter, -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Roboto, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;, &quot;Segoe UI Symbol&quot;; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><li style="box-sizing: border-box;">

Software Version | RP2772
-- | --
Hardware Version | WKE2.094.326A01
Device Model | HG6143D
Device Description | GPON
Serial Number | FHTT333333Q3
ONU State | O5(STATE_OPERATION)
ONU Regist State | OK
LOID | FiberHome

</li></ul>
</body>
</html>

But I came across the fact that with any settings I get Transmitted Power ----- Briefly when the status 05, Transmitted Power - 2 jumps. and again Transmitted Power ----- I understand correctly that the problem is in the calibration of the laser and on this model it is sewn into permanent memory, and you can change the calibration of the laser only after receiving newer software, where is the ability to download the calibration from the config available? Or am I mistaken ?

<html>
<body>

System
--
Device Name | IGD
Uptime | 42 min
Firmware Version | V1.0-220817
CPU Usage | 0%
Memory Usage | 49%

</body>
</html>
<html>
<body>

PON Status
--
Temperature | 54.898438 C
Voltage | 3.146900 V
Tx Power | 2.032531 dBm
Rx Power | -19.392485 dBm
Bias Current | 18.950000 mA

</body>
</html>
<html>
<body>

PON Status
--
Temperature | 54.558594 C
Voltage | 3.154700 V
Tx Power | -inf dBm
Rx Power | -19.207807 dBm
Bias Current | 0.000000 mA

</body>
</html>

**Comment by Ulkizz** [comment]
TX power jumping to 2 and back to inf - is absolute normal. It goes online, tries to authenticate, then falls back.

**Reply by Anime4000** [reply]
Yes, indicate OLT reject your PON Stick, best effort to update the firmware and properly clone all OMCI Info
